src/UI/Series/Details/InfoViewTemplate.hbs

```diff
@@ -55,6 +55,10 @@
             {{#if_eq seasonNumber compare="-1"}}
                 <span class="label label-default">{{title}}</span>
             {{/if_eq}}
+
+            {{#if_eq sceneSeasonNumber compare="-1"}}
```

Comment:
No, if one is populated the other will be null, at least in the current code (though I don't see that changing).